src/App.js

```diff
@@ -1,552 +1,57 @@
-                  <div className="gridLine" style={{gridRow: 27, gridColumn: 6}} />
-                  <div className="gridLine" style={{gridRow: 27, gridColumn: 7}} />
-                  <div className="gridLine" style={{gridRow: 28, gridColumn: 1}} />
-                  <div className="gridLine" style={{gridRow: 28, gridColumn: 2}} />
-                  <div className="gridLine" style={{gridRow: 28, gridColumn: 3}} />
-                  <div className="gridLine" style={{gridRow: 28, gridColumn: 4}} />
-                  <div className="gridLine" style={{gridRow: 28, gridColumn: 5}} />
-                  <div className="gridLine" style={{gridRow: 28, gridColumn: 6}} />
-                  <div className="gridLine" style={{gridRow: 28, gridColumn: 7}} />
-                  <div className="gridLine" style={{gridRow: 29, gridColumn: 1}} />
-                  <div className="gridLine" style={{gridRow: 29, gridColumn: 2}} />
-                  <div className="gridLine" style={{gridRow: 29, gridColumn: 3}} />
-                  <div className="gridLine" style={{gridRow: 29, gridColumn: 4}} />
-                  <div className="gridLine" style={{gridRow: 29, gridColumn: 5}} />
-                  <div className="gridLine" style={{gridRow: 29, gridColumn: 6}} />
-                  <div className="gridLine" style={{gridRow: 29, gridColumn: 7}} />
-                  <div className="gridLine" style={{gridRow: 30, gridColumn: 1}} />
-                  <div className="gridLine" style={{gridRow: 30, gridColumn: 2}} />
-                  <div className="gridLine" style={{gridRow: 30, gridColumn: 3}} />
-                  <div className="gridLine" style={{gridRow: 30, gridColumn: 4}} />
-                  <div className="gridLine" style={{gridRow: 30, gridColumn: 5}} />
-                  <div className="gridLine" style={{gridRow: 30, gridColumn: 6}} />
-                  <div className="gridLine" style={{gridRow: 30, gridColumn: 7}} />
-              </section>
-              </section>
-          </div>
-          </section>
+    <nav className='nav'>
+      {/* Top left */}
+      <section className='navTopLeft'>
+        <Link to='/' className='link logoButton'>
+            <div className='logoCircle'></div>
+            <h1>Meal and Mingle</h1>
+        </Link>
+      </section>
+      {/* Top right */}
+      <section className='navTopRight'>
+        <Link to='/' className='link navButton'>
+            <h1>Home</h1>
+        </Link>
+        <Link to='/messages' className='link navButton'>
+            <h1>Messages</h1>
+        </Link>
+        <Link to='/profile' className='link navProfile'>
+          <button className='navButton'><h1>{props.name}</h1></button>
+          <button className='userIcon headerVariation' style={{backgroundColor: props.color}}>{props.initials}</button>
+        </Link>
       </section>
+    </nav>
+  )
+}
+
+function App() {
+  return (
+    <Router>
```

Comment:
This is outside the scope of workshop 4, but you should consider using layout pages and <Outlet/> to help you with routing. We go over this in workshop 5 and the assignment 5 resources will have more information.
Using layout pages and <Outlet/> will make your definition of routes much cleaner inside the `App` component. We generally want to have the App component ONLY deal with routes, and not also have HTML stuff.